### PR TITLE
chore: added the extra examples from academy to the python readme 

### PR DESCRIPTION
**Pull request by John-Memphis**
Search for "TO:DO" (no quotes) in the file. There's a part that needs to be double checked for clarity about what the words actually mean.

### CODE REVIEW

**Comment by idanasulinmemphis** [comment]
@yanivbh1 please review

File: README.md [hunk 1]
```diff
@@ -93,7 +93,112 @@ if __name__ == '__main__':
...
+    )
+```
+
+The connect function in the Memphis class allows for the connection to Memphis. Connecting to Memphis (cloud or open-source) will be needed in order to use any of the other functionality of the Memphis class. Upon connection, all of Memphis' features are available.
+
+What arguments are used with the Memphis.connect function change depending on the type of connection being made.
+
+For details on deploying memphis open-source with different types of connections see the [docs](https://docs.memphis.dev/memphis/memphis-broker/concepts/security).
+ 
+The default connection type (for cloud) is using a JWT token-based connection. Here is an example of using memphis.connect with a JWT token:
```
Comment (yanivbh1) [inline README.md:122]: @idanasulinmemphis @shay23b not sure I understand this comment. Also, the default connection method is password-based.

Comment (yanivbh1) [reply README.md:122]: @John-Memphis I dont see any example with `account id` for connecting with the Cloud

File: README.md [hunk 2]
```diff
@@ -104,64 +209,218 @@ await memphis.close()
...
-  storage_type=Storage.DISK, # Storage.DISK/Storage.MEMORY. Defaults to DISK
-  replicas=1, # defaults to 1
-  idempotency_window_ms=120000, # defaults to 2 minutes
-  send_poison_msg_to_dls=True, # defaults to true
-  send_schema_failed_msg_to_dls=True, # defaults to true
-  tiered_storage_enabled=False, # defaults to false
-  partitions_number=1, # defaults to 1 
-  dls_station="<station-name>" # defaults to "" (no DLS station) - If selected DLS events will be sent to selected station as well
-)
+**A station will be automatically created for the user when a consumer or producer is used if no stations with the given station name exist.**<br><br>
```
Comment (yanivbh1) [inline README.md:212]: @John-Memphis , please add a phrase on what a station is

**Review by yanivbh1 — COMMENTED** [review]
Comments inside

File: README.md (hunk 1 above)
Comment (John-Memphis) [reply README.md:122]: @yanivbh1 I'll add an example for that.
image
image

All of the repos say that the default connection is token based. I've only been using Memphis with open-source and deployed with docker--where password is the default. I was assuming it was a kubernetes thing or a memphis cloud thing that used tokens by default.

If that's not the case I'll go and change that in all of the repos.

File: README.md (hunk 2 above)
Comment (John-Memphis) [reply README.md:212]: 👍

**Comment by John-Memphis** [comment]
Synced with master, addressed comments from @yanivbh1 and changed some of the sentences throughout for a better reading experience. 

This readme update should be ready.

**Comment by idanasulinmemphis** [comment]
@John-Memphis for me that looks ok, @yanivbh1 please take a look as well

**Comment by yanivbh1** [comment]
@idanasulinmemphis looks great to me

**Pull request merged**